### ISSUE TITLE
TypeError: super() takes at least 1 argument (0 given)

### ISSUE THREAD

**Issue by filipkis** [iconfinder/pyvat#44]
We're getting the following errors for our that use `pyvat` on our CI server, but I'm unable to reproduce the failure locally. However, before even figuring out why VIES service is returning server error, the error thrown below seems to originate in wrongly used `super` call, though I don't see what exactly is wrong there

```
  File "/home/runner/.local/lib/python2.7/site-packages/pyvat/__init__.py", line 212, in check_vat_number
    country_code)
  File "/home/runner/.local/lib/python2.7/site-packages/pyvat/registries.py", line 133, in check_vat_number
    raise ServerError(fault_code)
  File "/home/runner/.local/lib/python2.7/site-packages/pyvat/exceptions.py", line 3, in __init__
    super().__init__("ServerError: {}".format(fault_code))
TypeError: super() takes at least 1 argument (0 given)
```

Any idea what's happening here?

**Comment by JanmanX** [comment]
Hi @filipkis,

The issue was that the code used syntax used by Python 3 only. I have patched it in #45 and released a new version of PyVAT 1.3.15. 
Can you verify that it has fixed the issue?

**Comment by filipkis** [comment]
Thanks for a prompt reply and fix. 

It does seem to work now.
